### ISSUE TITLE
Build apk with flet_video doesnt works

### ISSUE THREAD

**Issue by Navnica** [flet-dev/flet#2968]
When I try to build a project using 

 ```powershell
flet build apk --include-packages flet_video
```
I'm getting something like this

```powershell
Creating Flutter bootstrap project...OK
Customizing app icons and splash images...OK
Generating app icons...OK
Generating splash screens...OK
Packaging Python app...OK
Building .apk for Android...Resolving dependencies...
  file_picker 6.2.1 (8.0.0+1 available)
  fl_chart 0.65.0 (0.67.0 available)
  flet 0.20.2 (0.21.2 available)
  flet_video 0.20.2 (0.21.2 available)
  flutter_lints 2.0.3 (3.0.2 available)
  flutter_native_splash 2.3.7 (2.4.0 available)
  js 0.6.7 (0.7.1 available)
  leak_tracker 10.0.0 (10.0.5 available)
  leak_tracker_flutter_testing 2.0.1 (3.0.5 available)
  leak_tracker_testing 2.0.1 (3.0.1 available)
  lints 2.1.1 (3.0.0 available)
  material_color_utilities 0.8.0 (0.11.1 available)
  meta 1.11.0 (1.12.0 available)
  package_info_plus 4.2.0 (6.0.0 available)
  petitparser 5.4.0 (6.0.2 available)
  screen_brightness 0.2.2+1 (1.0.0 available)
  screen_brightness_android 0.1.0+2 (1.0.0 available)
  screen_brightness_ios 0.1.0 (1.0.0+1 available)
  screen_brightness_macos 0.1.0+1 (1.0.0 available)
  screen_brightness_platform_interface 0.1.0 (1.0.0 available)
  screen_brightness_windows 0.1.3 (1.0.0 available)
  sensors_plus 4.0.2 (5.0.0 available)
  test_api 0.6.1 (0.7.0 available)
  toml 0.14.0 (0.15.0 available)
  vm_service 13.0.0 (14.2.0 available)
  xml 6.3.0 (6.5.0 available)
Got dependencies!
26 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.

Running Gradle task 'assembleRelease'...
Running Gradle task 'assembleRelease'...                           20,3s

┌─ Flutter Fix ──────────────────────────────────────────────────────────────────────────────────┐
│ [!] Your project requires a newer version of the Kotlin Gradle plugin.                         │
│ Find the latest version on https://kotlinlang.org/docs/releases.html#release-details, then     │
│ update C:\Users\Navnica\AppData\Local\Temp\flet_flutter_build_IiThjIIqbi\android\build.gradle: │
│ ext.kotlin_version = '<latest-version>'                                                        │
└────────────────────────────────────────────────────────────────────────────────────────────────┘

You are applying Flutter's app_plugin_loader Gradle plugin imperatively using the apply script method, which is deprecated and
will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block:
https://flutter.dev/go/flutter-gradle-plugin-apply

e: Incompatible classes were found in dependencies. Remove them from the classpath or use '-Xskip-metadata-version-check' to
suppress errors
e:
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/META
-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
e:
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/META
-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
e:
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/META
-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata
is 1.9.0, expected version is 1.7.1.
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (14, 10): Unresolved reference: listOf
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (19, 12): Unresolved reference: listOf
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (25, 12): Unresolved reference: listOf
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (26, 17): Unresolved reference: javaClass
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (55, 6): Class 'kotlin.Suppress' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Suppress.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (62, 12): Unresolved reference: listOf
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (78, 6): Class 'kotlin.Suppress' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Suppress.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (85, 12): Unresolved reference: listOf
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (90, 2): Class 'kotlin.Suppress' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Suppress.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (95, 52): Unresolved reference: let
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (96, 37): Unresolved reference: it
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (100, 52): Unresolved reference: let
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (101, 34): Unresolved reference: it
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (108, 5): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (110, 16): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (111, 9): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (114, 16): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (115, 9): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (117, 21): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (129, 38): Unresolved reference: lazy
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (133, 6): Class 'kotlin.Suppress' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Suppress.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (135, 7): Unresolved reference: run
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (137, 9): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (138, 19): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (142, 13): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (143, 19): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (144, 25): Unresolved reference: listOf
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (148, 19): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (151, 19): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (154, 7): Unresolved reference: run
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (156, 9): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (157, 19): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (159, 13): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (160, 25): Unresolved reference: listOf
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (164, 19): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Messages.g.kt: (167, 19): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of
its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Wakelock.kt: (23, 5): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Wakelock.kt: (24, 37): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\Wakelock.kt: (26, 23): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its
metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (16, 21): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary
version of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (17, 5): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version
of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (21, 21): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary
version of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (22, 5): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version
of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (26, 15): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary
version of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (30, 15): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary
version of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (34, 5): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version
of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (38, 5): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version
of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class
e:
C:\Users\Navnica\AppData\Local\Pub\Cache\hosted\pub.dev\wakelock_plus-1.2.1\android\src\main\kotlin\dev\fluttercommunity\plus\wak
elock\WakelockPlusPlugin.kt: (42, 16): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary
version of its metadata is 1.9.0, expected version is 1.7.1.
The class is loaded from
C:/Users/Navnica/.gradle/caches/transforms-3/439afb987356d2e6a8cedea75b224908/transformed/jetified-kotlin-stdlib-1.9.23.jar!/kotl
in/Unit.class

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':wakelock_plus:compileReleaseKotlin'.
> A failure occurred while executing
org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 19s
Gradle task assembleRelease failed with exit code 1

Error building Flet app - see the log of failed command above.
```

I tried to reset to flat==0.20.0, put the latest version of Kotlin in my environment variables, the latest version of Gradle, flutter. There is no effect.

If I try to use the build without ```--include-packages flet_video```, then there are no errors at alll and I get a working apk file

Windows 11, Python 3.12

**Comment by syleishere** [comment]
https://github.com/flet-dev/flet/issues/3034

**Comment by FeodorFitsner** [comment]
It is fixed in Flet 0.23.1.

Please cleanup Cookiecutter cache before running `flet build` command.

On Windows:

```
rd /s /q %USERPROFILE%\.cookiecutters
```

On Mac/Linux:

```
rm -rf ~/.cookiecutters
```
